manufacturing/filter_holder/README.md

```diff
@@ -65,6 +65,14 @@ tubing, and the other - with vents for air in/out-flow.
 
 Most recent changes also include thickening of the walls to make the 3d prints less fragile.
 
+### Lower Face Plate
+
+This is a clear acrylic sheet that houses the filter assembly. It is mounted using heat inserts. Link to the part file is given in [Source files](#Source-files) below. Some other useful files for the same part are [.idw 2D drawing](CAD/LowerAcrylicFacePlate.idw), [.pdf 2D drawing](CAD/LowerAcrylicFacePlate.pdf),[.stl for 3D printing](CAD/LowerAcrylicFacePlate.stl),[.dxf of the front face for laser cutting](CAD/LowerAcrylicFacePlate_FrontFace.dxf).
+
+![](CAD/LowerAcrylicFacePlate.png)
+
+The sizing for the mounting holes was based on [this](https://www.pemnet.com/fastening_products/pdf/sidata.pdf) catalog for heat set insert
```

Comment:
This is not really sufficient information to understand the decision. There are many types listed in that catalog. Which precise type are we using, on which page and line, and what is the conclusion of that decision?

We should probably provide the same analysis for the decision on the diameter of the holes for heat inserts in the filter housings themselves.

Reply:
I have provided some more information about the mounting hole heat insert selection. And provided specifics on where to find the part in the document.